New translation: custom-html.md (#133)
gatsbyjs/gatsby-pt-BR: docs/docs/custom-html.md.md

> 

Reviewer: ```suggestion
---
```

Author: Alteração

> titulo: Personalizar html.js

Reviewer: ```suggestion
title: Personalizar html.js
```

Author: Alteração título

> 

Reviewer: ```suggestion
---
```

Author: Alteração

> Gatsby utiliza um componente React para a renderização do servidor '<head>' e outras partes do HTML fora do aplicativo principal do Gatsby. Além diso, o Gatsby define um valor padrão do ´<noscript>´ marcado aqui.

Reviewer: ```suggestion
Gatsby utiliza um componente React para a renderização do servidor '<head>' e outras partes do HTML fora do aplicativo principal do Gatsby. Além disso, o Gatsby define um valor padrão do ´<noscript>´ marcado aqui.
```

> Gatsby utiliza um componente React para a renderização do servidor '<head>' e outras partes do HTML fora do aplicativo principal do Gatsby. Além disso, o Gatsby define um valor padrão do ´<noscript>´ marcado aqui.

Reviewer: ```suggestion
Gatsby utiliza um componente React para a renderização do servidor `<head>` e outras partes do HTML fora do aplicativo principal do Gatsby.
```

Reviewer: Removi uma parte que não existe no [documento original](https://raw.githubusercontent.com/gatsbyjs/gatsby-pt-BR/master/docs/docs/custom-html.md)

> **A maioria dos sites deveria utilizar o `html.js` padrão enviado com o Gatsby. Contudo, se você precisar customizar seu site em html.js, copie o padrão em sue código o executando:

Reviewer: ```suggestion
**A maioria dos sites deveria utilizar o `html.js` padrão enviado com o Gatsby. Contudo, se você precisar customizar seu site em html.js, copie o padrão em seu código o executando:
```

> **A maioria dos sites deveria utilizar o `html.js` padrão enviado com o Gatsby. Contudo, se você precisar customizar seu site em html.js, copie o padrão em seu código o executando:

Reviewer: ```suggestion
A maioria dos sites deveria utilizar o `html.js` padrão enviado com o Gatsby. Contudo, se você precisar customizar seu site em html.js, copie o padrão em seu código o executando:
```

Author: Alteração **

> A maioria dos sites deveria utilizar o `html.js` padrão enviado com o Gatsby. Contudo, se você precisar customizar seu site em html.js, copie o padrão em seu código o executando:

Reviewer: ```suggestion
A maioria dos sites deveria utilizar o `html.js` padrão enviado com o Gatsby. Contudo, se você precisar customizar o html.js de seu site, copie o padrão em seu código executando:
```

```shell
cp .cache/default-html.js src/html.js
```
E depois faça as modificações que você necessita.

> 

Reviewer: ```suggestion
Se você precisar adicionar html customizado no `<head>` ou `<footer>` em cada página de seu site, você pode utilizar `html.js`.

> Personalizar o `html.js` é uma alternativa para quando o uso das APIs apropriadas não estiverem disponíveis no `gatsby-ssr.js`. Considere usar [`onRenderBody`](/docs/ssr-apis/#onRenderBody) or [`onPreRenderHTML`](/docs/ssr-apis/#onPreRenderHTML) ao invés do método acima.
> Como consideração adicional, a customização do `html.js` não é suportada dentro de um Tema Gatsby. Use o método de API mencionado acima.
```

Reviewer: Existe esse bloco no [documento original](https://github.com/gatsbyjs/gatsby-pt-BR/blob/master/docs/docs/custom-html.md)

## Props Necessárias

> Note: Os vários acessórios que são renderizados na página são necessários, por exemplo: 

Reviewer: ```suggestion
Note: Os vários _props_ que são renderizados na página são necessários, por exemplo: 
```

> Note: Os varios acessórios que são renderizados na página são necessários, por exemplo: 

Reviewer: ```suggestion
Note: Os vários acessórios que são renderizados na página são necessários, por exemplo: 
```

`headComponents`, `preBodyComponents`, `body`, e `postBodyComponents`. 

> ## Inserindo HTML no ´<head>´

Reviewer: ```suggestion
## Inserindo HTML no `<head>`
```

> Qualquer coisa que você renderizar no `html.js`, o componente não será "ativo" no cliente como outros componentes. Se

Reviewer: ```suggestion
Qualquer coisa que você renderizar no componente `html.js` não será "ativo" no cliente como outros componentes. Se
```

> Qualquer coisa que você renderizar no `html.js`, o componente não será "ativo" em o cliente como outros componentes. Se

Reviewer: ```suggestion
Qualquer coisa que você renderizar no `html.js`, o componente não será "ativo" no cliente como outros componentes. Se
```

> você quiser tornar mais dinâmica a atualização de sua ´<head>´, nós recomendamos o uso de: 

Reviewer: ```suggestion
você quiser tornar mais dinâmica a atualização de sua `<head>`, nós recomendamos o uso de: 
```

> você quiser torna-lo mais dinâmico atualiza a sua ´<head>´, nos recomendamos o uso de: 

Reviewer: ```suggestion
você quiser torná-lo mais dinâmico atualiza a sua ´<head>´, nós recomendamos o uso de: 
```

> você quiser torná-lo mais dinâmico atualiza a sua ´<head>´, nós recomendamos o uso de: 

Reviewer: ```suggestion
você quiser tornar mais dinâmica a atualização de sua ´<head>´, nós recomendamos o uso de: 
```

[React Helmet](/packages/gatsby-plugin-react-helmet/)

> ###Inserir HTML no `<footer>`

Reviewer: ```suggestion
### Inserir HTML no `<footer>`
```

Author: Alteração- espaçamento

> ### Inserir HTML no `<footer>`

Reviewer: ```suggestion
## Inserir HTML no `<footer>`
```

> Se você quiser inserir html personalizado no 'footer/rodapé', o caminho preferido para se fazer isso é pelo 'html.js'. Se você está escrevendo um plugin, considere o uso do  `setPostBodyComponents` suporte no [Gatsby SSR API](/docs/ssr-apis/).

Reviewer: ```suggestion
Se você quiser inserir html personalizado no _footer_, o caminho preferido para se fazer isso é pelo 'html.js'. Se você está escrevendo um plugin, considere o uso do da prop  `setPostBodyComponents` no [Gatsby SSR API](/docs/ssr-apis/).
```

> ***###Target container 

Reviewer: ```suggestion
### Target container 
```

> ### Target container 

Reviewer: ```suggestion
## Target container 
```

> Se você ver o erro: `Uncaught Error: _registerComponent(...):Target container is not a DOM element.` isso significa que o seu ´<html.js>´ está com a ausência da solicitação "target container". Dentro do seu ´<body>´ você deve ter uma div com o id `___gatsby` como:

Reviewer: ```suggestion
Se você ver o erro: `Uncaught Error: _registerComponent(...):Target container is not a DOM element.` isso significa que o seu `<html.js>` está com a ausência da solicitação "target container". Dentro do seu `<body>` você deve ter uma div com o id `___gatsby` como:
```

> Se você ver o erro: `Uncaught Error: _registerComponent(...):Target container is not a DOM element.` isso significa que o seu ´<html.js>´ está coma ausência da solicitação "target container". Dentro do seu ´<body>´ você deve ter um div com uma IDE `___gatsby` como:

Reviewer: ```suggestion
Se você ver o erro: `Uncaught Error: _registerComponent(...):Target container is not a DOM element.` isso significa que o seu ´<html.js>´ está com a ausência da solicitação "target container". Dentro do seu ´<body>´ você deve ter um div com uma IDE `___gatsby` como:
```

> Se você ver o erro: `Uncaught Error: _registerComponent(...):Target container is not a DOM element.` isso significa que o seu ´<html.js>´ está com a ausência da solicitação "target container". Dentro do seu ´<body>´ você deve ter um div com uma IDE `___gatsby` como:

Reviewer: ```suggestion
Se você ver o erro: `Uncaught Error: _registerComponent(...):Target container is not a DOM element.` isso significa que o seu ´<html.js>´ está com a ausência da solicitação "target container". Dentro do seu ´<body>´ você deve ter uma div com o id `___gatsby` como:
```

```jsx:title=src/html.js
<div
  key={`body`}
  id="___gatsby"
  dangerouslySetInnerHTML={{ __html: this.props.body }}
/>
```

> ###Adicionado o JavaScript personalizado

Reviewer: ```suggestion
### Adicionado o JavaScript personalizado
```

> ### Adicionado o JavaScript personalizado

Reviewer: ```suggestion
## Adicionado o JavaScript personalizado
```

> Você pode adicionar JavaScript personalizado ao seu documento HTML usando o atributo [dangerouslySetInnerHTML] (https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) do React.

Reviewer: ```suggestion
Você pode adicionar JavaScript personalizado ao seu documento HTML usando o atributo [dangerouslySetInnerHTML] (https://pt-br.reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) do React.
```
